sig/datadog/appsec/security_engine/runner.rbs

```diff
@@ -12,11 +12,19 @@ module Datadog
 
         @waf_context: WAF::Context
 
+        @ruleset_version: ::String
+
+        @waf_addresses: ::Array[::String]
```

Comment:
I think we should define it in a better place (responsible for addresses) and reference it here. Otherwise the origin might change, but type check will not fail

Reply:
fixed in [f4de7ed](https://github.com/DataDog/dd-trace-rb/pull/4839/commits/f4de7edb068f62f5413a2678b9197d6063015c30)